pixelmatch.py

```diff
@@ -1,45 +1,105 @@
-DEFAULT_OPTIONS = {
-    "threshold": 0.1,  # matching threshold (0 to 1); smaller is more sensitive
-    "includeAA": False,  # whether to skip anti-aliasing detection
-    "alpha": 0.1,  # opacity of original image in diff ouput
-    "aa_color": [255, 255, 0],  # color of anti-aliased pixels in diff output
-    "diff_color": [255, 0, 0],  # color of different pixels in diff output
-    "diff_mask": False,  # draw the diff over a transparent background (a mask)
-}
+import functools
+from typing import Sequence, Union, List, Tuple
+
+
+ImageData = List[int]
+PILImageOrImageArray = Union["Image", Sequence[float]]
+RGBTuple = Tuple[int, int, int]
```

Comment:
I thought about this but opted for a stricter definition so that you don't get something like [1,2,3,4] and your IDE can't warn you because it fits in the type definition of List[int] ~~I think that one could use the new 3.8 exclusive `Literal` type.~~ That wouldn't work either.

Most people messing around with images would be smart enough to know where the error is coming from if they provided an `A` value, and I guess if we broaden this the IDE (correctly) won't complain about `[1,1,1]`.